domain/personallearningplan/src/test/kotlin/uk/gov/justice/digital/hmpps/domain/personallearningplan/service/GoalServiceTest.kt

```diff
@@ -424,14 +424,35 @@ class GoalServiceTest {
       verify(goalPersistenceAdapter).getGoals(prisonNumber)
     }
 
+    @Test
+    fun `should return all goals given empty filter`() {
+      // Given
+      given(goalPersistenceAdapter.getGoals(any())).willReturn(listOf(activeGoal, archivedGoal, completedGoal))
+
+      // When
+      val actual = service.getGoals(GetGoalsDto(prisonNumber, emptySet()))
+
+      // Then
+      assertThat(actual).isEqualTo(
+        GetGoalsResult.Success(
+          listOf(
+            activeGoal,
+            archivedGoal,
+            completedGoal,
+          ),
+        ),
+      )
+      verify(goalPersistenceAdapter).getGoals(prisonNumber)
+    }
+
     @ParameterizedTest
     @EnumSource(GoalStatus::class)
-    fun `should return only goals matching status if a filter is set`(status: GoalStatus) {
+    fun `should return only goals matching status if a filter is set with a single status`(status: GoalStatus) {
```

Comment:
Is there a unit test for multiple?

Reply:
Good spot; I thought I'd written one, but clearly not 🤣 
I'll add one 👍